src/solvers/mod.rs

```diff
@@ -1,3 +1,28 @@
+//! This module provides the interface to different solvers.
+//!
+//! Both [`coin_cbc`](https://docs.rs/coin_cbc/latest/coin_cbc/) and
+//! [`minilp`](https://docs.rs/minilp/0.2.2/minilp/) are available as cargo
+//! [features](https://doc.rust-lang.org/cargo/reference/features.html). To use
+//! them, specify your dependency to `lp_modeler` accordingly in your `Cargo.toml`
+//! (note the name difference of the `native_coin_cbc` feature for the `coin_cbc` crate):
+//! ```ignore
```

Comment:
It looks weird to have this as ignored tests (both on the docs and running cargo test). Maybe it is common practice to set these as ignored to highlight them, I don't know. Wouldn't it be better to use ` ```toml`?

Reply:
Ah, cool. I didn't know I could do this, thanks for pointing it out. Done and pushed...